media/windows/ricoh-printer/ricoh_db_create.py

```diff
@@ -0,0 +1,121 @@
+#!/user/bin/env python3
+
+# Script which runs daily to update staff print totals
+# based on CSVs generated by the printer itself.
+
+import sys
+sys.path.insert(1, '../../python/')
+
+import os
+import re
```

Comment:
I don't think you're using `re`, so you might as well remove this import.  It's probably a good idea to remove all imports that aren't being used.